src/controllers/lyrics.js

```diff
@@ -69,6 +72,9 @@ export default function (view) {
         if (lyric) {
             lyric.classList.remove('pastLyric');
             lyric.classList.remove('futureLyric');
+            if (autoScroll) {
+                scrollManager.scrollToElement(lyric, true);
```

Comment:
It now works when switching between views, but we need to use instant scrolling on `viewshow`.
It currently scrolls smoothly from top do bottom.

We could use a boolean (say `smoothScroll`) (set `false` in `viewshow`, set `true` here).
Or "enum" (state) for `autoScroll`: `noscroll = 0`, `instant = 1`, `smooth = 2`.

Reply:
I am partial to using Typscript enums for this. Where should I place the typescript file? In types or in the same folder as the lyrics.js